.gitignore

```diff
@@ -1,3 +1,4 @@
 .publish/
 **/coverage.json
 **/testreport.json
+coverage.json
```

Comment:
This was merged in a separate PR 😅
```suggestion
```

Reply:
Yes, you did it right after I maybe this change in this PR :) Guess we spotted the same problem.